src/polyswarm/client/utils.py

```diff
@@ -20,7 +20,8 @@ def parse_hashes(hashes, hash_file=None):
     hashes = list(hashes)
     if hash_file is not None:
         hashes += hash_file.readlines()
-    return hashes
+
+    return [h.strip('\n') for h in hashes]
```

Comment:
How big will be this list?
Or: Should it be a Generator, or an full-in-memory list is ok here?

Reply:
This is a concern indeed, but addressing really large lists is something we will postpone rn. I believe we can assume it fits in memory. This can be a future improvement though if we really want to handle extremely large files.